_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+
+Nick listens more than he speaks. I will advise to not disturb him if he is
+coding. He enjoys being concentrated while coding. It was the great experience
+of getting his mentorship. He is contributing to [Core
+Python](https://github.com/python/cpython) for a few decades now.  You will
+enjoy discussing on interesting code compositions with him. He is a Red Hatter.
+
+#### [Praveen Patil](https://twitter.com/_gnovi)
+
+Unfortunately, I didn't get much time to talk with Praveen during this
+conference. He is a Math teacher who teaches concepts of Mathematics using
+Python programming language. You should feel confident to speak with him on
+Python in Education and Mathematics with him.
+
+
+#### [John Hawley](https://github.com/warthog9)
+
+John is the wittiest person in my known. His lines always end with a humor. He
+hacks mostly on hardware and the GNU/Linux. Micro Python and GNU/Linux should be
+considered as his part of interests.
+
+I am sad to declare that I was unable to attend any keynote speeches because of
+the illness. Mostly I rested at the hotel or talked with people during the
+conference days.
+
+
+![pycon_pune_2017_keynote_speackers]({{
+site.url}}/assets/images/pycon_pune_2017_keynote_speackers.jpg)
+
+
+#### Volunteer Party
+
+If you are volunteering for this conference, then you will be invited to a
+volunteer dinner party. We enjoyed party colored disco lights dancing on the
+bits of the DJ. The Punjabi food and if you are above 25 than you were allowed
+to take a sip of a beer.
+
+#### Devsprint
+
+Devsprint happen at the [Redhat Headquarters,
+Pune](https://goo.gl/maps/mXeirzQhPFz). I found the building has tight security.
+You will find an individual pantry section dedicated to each department. We were
+instructed to hack at huge cafeteria section. I myself contributed to Core
+Python. Nick Coghlan was mentoring for Core Python. I managed to review one PR,
+find one broken test case and written fix of an existing issue with the help of
+him. Honza was leading the development of Django web framework. A team of [Anand
+Chitipothu](http://anandology.com/) mentored for [Web2py](www.web2py.com/).
+[Farhaan Bukhsh](https://twitter.com/fhackdroid) mentored for
+[Pagure](https://github.com/pypingou/pagure). John Hawley encouraged on
+contributing to [MicroPython](https://micropython.org/). Terr Oda, Stephen
+Turnbull and Florian Fuchs mentored for
+[GNU/Mailman](https://en.wikipedia.org/wiki/GNU_Mailman).
+
+
+![cpython_devsprint]({{
+site.url}}/assets/images/cpython_devsprint.jpg)
+
+#### Why to attend this conference?
+
+* This conference has coolest Devsprint. The organizers understand the value of
+  the Devsprint in a conference. I have never observed such importance of
+  Devsprint at any other Python conference happening in India.
+
+* If you are a student, then this is the beginner friendly conference. You
+  should not be a worry to attend if you are a Python noobs. You will receive a
```

Comment:
If you are a student, then this is *a* beginner friendly conference. Don't be afraid to attend if you are a Python noob.